### ISSUE TITLE
Communicate use of flexible csv and new fields

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1537]
With the deployment of the flexible csv and the new fields 
We'll need to decide on a communucation plan - bearing in mind that ony spreadsheet users can currently use it, not WinREG.

**Comment by DeniseColbert** [comment]
Plan started, shared with Kirk, Alison, Eric and Steve. Link: https://docs.google.com/document/d/1yFSrrrWrkm6-tUKETs4KzEMvbpiYPDzkJWIUnB1Kbe0/edit?usp=sharing

**Comment by richpomfret** [comment]
Limited in how we proceed with this.

**Comment by edickens** [comment]
I'm a bit loathe to get transcribers onto this until we have WinFreeREG.  It may tempt transcribers to move to a spreadsheet and this is a backward step.  WinFreeREG is nearly ready to take the flexible csv.

**Comment by SteveBiggs** [comment]
Good point Eric. If any existing spreadsheet users ask , I'll give them details but won't broadcast anything yet.

**Comment by edickens** [comment]
Close this?  Will be superseded by online data entry.

**Comment by SteveBiggs** [comment]
It has been in the Transcriber's help on the website for some time and most coords and transcribers are now aware of it, but I could send out a message if we feel it necessary?

**Comment by edickens** [comment]
The new fields are in the Help.  When someone needs to use a new field we do get them to use flexible CSV.  But many transcribers are on WinReg so would have to move to spreadsheet entry.

**Comment by edickens** [comment]
Transcribers really do want this, especially BA to get the surname correct.  There have been exchanges in the Transcribers mailgroup these last few days on the subject.

**Comment by edickens** [comment]
Can be closed.
